signer/README.md

```diff
@@ -17,6 +17,8 @@ You can run the signer by providing the required environment variables.
 If not specified, the default values from `./src/config/default.toml` will be used.
 - `SIGNER_BLOCKLIST_CLIENT__HOST`=<server-hostname-or-ip>
 - `SIGNER_BLOCKLIST_CLIENT__PORT`=<server-port>
+- `SIGNER_STACKS_API_ENDPOINT`=<schema-and-fqdn-and-port>
+- `SIGNER_STACKS_NODE_ENDPOINT`=<schema-and-fqdn-and-port>
```

Comment:
Wouldn't we get a double underscore before `ENDPOINT` here like the previous env variables?

Reply:
We don't need it in these cases, since each part of the config entry doesn't contain an underscore itself. If it did, things would be ambiguous so we'd need a double underscore (or something else) to disambiguate the path in the config.